blocks/tabs/tabs.css

```diff
@@ -41,8 +41,8 @@
 
 
 .tabs .tabs-list button[aria-selected='true'] {
-  border-bottom: 1px solid var(--background-color);
-  background-color: var(--background-color);
+  border-bottom: 1px solid var(--color-background-default);
```

Comment:
Let's change the `color` and `background-color` of the `.tabs .tabs-list button` selector on lines 27 and 28 to change with the theme.